README.md

```diff
@@ -21,127 +47,131 @@ c2patool is a command line tool for working with C2PA [manifests](https://c2pa.o
-c2patool sample/C.jpg -d
+c2patool -d sample/image.jpg
 ```
 
-### Adding a manifest to a file
-
-To add C2PA data to a file, use the `-m` or `--manifest` option and provide a manifest definition JSON file as the argument. The tool generates a new manifest using the values given in the file.
-
-The file path must reference the asset to be signed. 
+The tool displays the detailed report to standard output (stdout).
 
-The output file is specified on the command line via the `-o` or `--output` flag.  If the output is the same as the source, it will be overwritten. Use this with caution. If no output is given you can preview the generated manifest but nothing is written.
+### Adding a manifest to an asset file
 
-The generated manifest store will also be reported in JSON format to stdout.
+To add C2PA manifest data to a file, use the `--manifest` / `-m` option with a manifest JSON file as the option argument and the path to the asset file to be signed. Specify the output file as the argument to the `--output` / `-o` option. For example:
 
 ```shell
 c2patool sample/image.jpg -m sample/test.json -o signed_image.jpg
 ```
 
-A parent file can be specified with the `-p` or `--parent` option or in the manifest definition. The parent file represents the state of the asset before any edits were made. 
+The tool generates a new manifest using the values given in the file and displays the manifest store to standard output (stdout).
+
+CAUTION: If the output file is the same as the source file, the tool will overwrite the source file. 
+
+If you do not use the `--output` / `-o` option, then the tool will display the generated manifest but will not save it to a file.
+
+#### Specifying a parent file
+
+A _parent file_ represents the state of the image before any edits were made. 
+
+Specify a parent file as the argument to the `--parent` / `-p` option; for example:
 
 ```shell
 c2patool sample/image.jpg -m sample/test.json -p sample/c.jpg -o signed_image.jpg
 ```
 
+You can also specify a parent file in the manifest definition.
+
 #### Forced overwrite
-The tool will return an error if the output file already exists. The `-f` or `--force` flag may be used to override this behavior. Use this with caution.
+
+The tool will return an error if the output file already exists. Use the `--force` / `-f` option to force overwriting the output file. For example:
 
 ```shell
 c2patool image.jpg -m sample/test.json -f -o signed_image.jpg
 ```
 
-#### Manifest preview feature
+### Previewing a manifest
 
-If the output file is not specified, the tool will generate a preview of the generated manifest. This can be used to make sure you have formatted the manifest definition correctly.
+To display a preview of the generated manifest and ensure you've formatted the manifest definition correctly, provide the path to a manifest file as the argument with no other options or flags; for example:
 
 ```shell
-c2patool image.jpg -m sample/test.json
+c2patool sample/test.json
```

Comment:
You always need to specify the asset you are acting on. 
c2patool sample/image.jpg -m sample/test.json